image.go

```diff
@@ -10,9 +12,16 @@ type ImageInfo struct {
 	Name string
 }
 
+type SaveOptions struct {
+	Platforms []specs.Platform
```

Comment:
suggestion: keep details private so changes can be used without breaking the API.

Reply:
The reason I need it is for the K3S module so I would have to implement a duplicate type in that module

Reply:
Actually, I made it work while keeping it private.